skimage/feature/blob.py

```diff
@@ -141,22 +156,27 @@ def _prune_blobs(blobs_array, overlap):
     overlap : float
         A value between 0 and 1. If the fraction of area overlapping for 2
         blobs is greater than `overlap` the smaller blob is eliminated.
+    sigma_dim : int, optional
+        The number of columns in ``blobs_array`` corresponding to sigmas rather
+        than positions.
 
     Returns
     -------
     A : ndarray
         `array` with overlapping blobs removed.
     """
-    sigma = blobs_array[:, -1].max()
-    distance = 2 * sigma * sqrt(blobs_array.shape[1] - 1)
-    tree = spatial.cKDTree(blobs_array[:, :-1])
+    sigma = blobs_array[:, -sigma_dim].max()
```

Comment:
shouldn't this be `sigma = blobs_array[:, -sigma_dim:].max()` ?

Reply:
Oops! Yes! Fixed!

Reply:
Very nice catch btw. ;)